cmd/utils.go

```diff
@@ -2113,3 +2114,107 @@ func checkValidLicense(log *LoggingConfig, license string) error {
 }
+func lintMountPathForSingleFile(path string, log *LoggingConfig) {
+
+	file, err := os.Stat(path)
+	if err != nil {
+		log.Warning.logf("Could not stat mount path: %s", path)
+
+	} else {
+		if file.Mode().IsDir() {
+			log.Debug.logf("Path %s for mount is a directory", path)
+		} else {
+
+			log.Warning.logf("Path %s for mount points to a single file.  Single file Docker mount paths cause unexpected behavior and will be deprecated in the future.", path)
+		}
+
+	}
+}
+
+func lintMountVar(mountListSource string, log *LoggingConfig, stackPath string) (int, int) {
+
+	if mountListSource == "" {
+		log.Warning.log("No APPSODY MOUNTS exists, mount paths can not be validated.")
+		return 0, 1
+	}
+	errCount := 0
+	warningCount := 0
+	mountList := strings.Split(mountListSource, ";")
+
+	templatePath := filepath.Join(stackPath, "templates")
+	log.Debug.log("Checking for template path: ", templatePath)
+	fileCheck, err := Exists(templatePath)
+	log.Debug.log("Template path exists: ", fileCheck)
+	if err != nil {
+		log.Error.log("Error attempting to determine if template path exists: ", err)
+		return 0, 1
+	}
+	if !fileCheck {
+		log.Error.log("Missing template directory in: ", stackPath)
+		return 0, 1
+	}
+	if IsEmptyDir(templatePath) {
+		log.Error.log("No templates found in: ", templatePath)
+		return 0, 1
+	}
+	templates, _ := ioutil.ReadDir(templatePath)
+
+	// loop through the template directories
+	for _, f := range templates {
+
+		for _, mount := range mountList {
+			traceMount := strings.Trim(mount, "\"")
+			log.Debug.log("mount pair: ", traceMount)
+			localPaths := strings.Split(traceMount, ":")
+			if len(localPaths) != 2 {
+				log.Error.log("Mount is not properly formatted: ", traceMount)
+				errCount++
+			} else {
+
+				localPath := localPaths[0]
+				if localPath == "" {
+					log.Error.logf("Path for mount %s is empty: ", traceMount)
+					errCount++
+				} else {
+
+					log.Debug.log("local path: ", localPath)
+					if localPath == "/" || localPath == "." {
+						log.Debug.logf("Path %s for mount %s is a directory.", localPath, traceMount)
+					} else if localPath[0:1] == "~" {
+						log.Debug.logf("Path %s for mount %s can not be evaluated at this time.", localPath, traceMount)
+					} else {
+						mountFilePath := filepath.Join(stackPath, "templates", f.Name(), localPath)
+
+						log.Debug.log("mountFilePath: ", mountFilePath)
+						warnings, mountLintError := validateMountPath(mountFilePath, traceMount, log)
+						warningCount = warningCount + warnings
+						if mountLintError != nil {
+							errCount++
+						}
+
+					}
+				}
+			}
+		}
+	}
+	return warningCount, errCount
+}
+func validateMountPath(path string, mount string, log *LoggingConfig) (int, error) {
+	log.Debug.log("Attempting to validate mount path: ", path)
+	var mountError error
+	warningCount := 0
+	file, err := os.Stat(path)
+	if err != nil {
+
+		mountError = errors.Errorf("Could not stat path: %s for mount %s", path, mount)
```

Comment:
This should be printed somewhere. Maybe print here and return a count instead of error